test/models/user_test.rb

```diff
@@ -3,6 +3,8 @@
 class UserTest < ActiveSupport::TestCase
   should validate_presence_of :email
   should validate_uniqueness_of :email
+  should_not allow_value("hi").for(:email)
+  should allow_value("john@example.nl").for(:email)
```

Comment:
Just to be sure for the future, can we also add a test for this cases:

```ruby
should allow_values"john+doe@example.com").for(:email)
should allow_values"john@example.city").for(:email)
```

(Include the `+` case, and some weird exotic tld)